dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/interceptor/OneWayProcessorInterceptor.java

```diff
@@ -0,0 +1,181 @@
+                //the transport might discard any data on the stream when this
+                //thread unwinds or when the empty response is sent back
+                DelegatingInputStream in = message.getContent(DelegatingInputStream.class);
+                if (in != null) {
+                    in.cacheInput();
+                }
+            }
+
+            if (robust) {
+                // continue to invoke the chain
+                chain.pause();
+                chain.resume();
+                if (message.getContent(Exception.class) != null) {
+                    // CXF-5629 fault has been delivered alread in resume()
+                    return;
+                }
+            }
+
+            try {
+                Message partial = createMessage(message.getExchange());
+                partial.remove(Message.CONTENT_TYPE);
+                partial.setExchange(message.getExchange());
+                Conduit conduit = message.getExchange().getDestination()
+                    .getBackChannel(message);
+                if (conduit != null) {
+                    message.getExchange().setInMessage(null);
+                    //for a one-way, the back channel could be
+                    //null if it knows it cannot send anything.
+                    conduit.prepare(partial);
+                    conduit.close(partial);
+                    message.getExchange().setInMessage(message);
+                }
+            } catch (IOException e) {
+		LOG.finest("OneWayProcessorInterceptor: Ignoring IOException 2: " + e); // Liberty Change
+                //IGNORE
+            }
+
+            if (!useOriginalThread && !robust) {
+                chain.pause();
+                try {
+                    final Object lock = new Object();
+                    synchronized (lock) {
+                        message.getExchange().getBus().getExtension(WorkQueueManager.class)
+                            .getAutomaticWorkQueue().execute(new Runnable() {
+                                public void run() {
+                                    synchronized (lock) {
+                                        lock.notifyAll();
+                                    }
+                                    chain.resume();
+                                }
+                            });
+                        //wait a few milliseconds for the background thread to start processing
+                        //Mostly just to make an attempt at keeping the ordering of the
+                        //messages coming in from a client.  Not guaranteed though.
+                        lock.wait(20L);
+                    }
+                } catch (RejectedExecutionException e) {
+                    LOG.warning(
+                        "Executor queue is full, run the oneway invocation task in caller thread."
+                        + "  Users can specify a larger executor queue to avoid this.");
```

Comment:
Should this have a Liberty change flag?

Reply:
No, this warning message exists in original CXF code.